Source/WebKit/UIProcess/API/wpe/qt6/WPEQtView.h

```diff
@@ -38,6 +39,7 @@ class Q_DECL_EXPORT WPEQtView : public QQuickItem {
     Q_PROPERTY(QString title READ title NOTIFY titleChanged)
     Q_PROPERTY(bool canGoBack READ canGoBack NOTIFY loadingChanged)
     Q_PROPERTY(bool canGoForward READ canGoForward NOTIFY loadingChanged)
+    Q_PROPERTY(WPEQtProfile* profile READ profile WRITE setProfile NOTIFY profileChanged FINAL)
```

Comment:
The `QWebEnginePage` class does _not_ have a `profile` property either, only a `::profile()` accessor method.

Reply:
Correct, however the QML exposed types do have the `profile` property and require them to be set-able.